packages/dev/loaders/src/glTF/2.0/Extensions/KHR_interactivity.ts

```diff
@@ -46,7 +47,7 @@ export class KHR_interactivity implements IGLTFLoaderExtension {
 
         const json = convertGLTFToSerializedFlowGraph(interactivityDefinition);
         const coordinator = new FlowGraphCoordinator({ scene });
-        FlowGraph.Parse(json, { coordinator, pathConverter: this._pathConverter });
+        FlowGraph.Parse(json, { coordinator, pathConverter: this._pathConverter! });

```

Comment:
The proper thing to do is check to make sure this is available at the beginning of this function since it could be falsy if someone disposes the asset mid load.